src/lib/drivers/extension.js

```diff
@@ -80,17 +80,34 @@ class ExtensionDriver extends Driver {
     this._listeners[eventName].push(cb);
   }
 
+  /**
+   * Bind a one-time listener for protocol events. Listener is removed once it
+   * has been called.
+   * @param {!string} eventName
+   * @param {function(...)} cb
+   */
+  once(eventName, cb) {
+    // Create a replacement listener that will immediately remove itself after calling cb once.
+    const cbGuard = function() {
```

Comment:
Just make this an arrow fn. No need to explicitly bind it since it's not going via `addEventListener` in a browser.